packages/engine/Source/Core/ITwinPlatform.js

```diff
@@ -37,6 +37,64 @@ ITwinPlatform.ExportType = Object.freeze({
   "3DTILES": "3DTILES",
 });
 
+/**
+ * Types of Reality data
+ * https://developer.bentley.com/apis/reality-management/rm-rd-details/#types
+ * @enum {string}
+ */
+ITwinPlatform.RealityDataType = Object.freeze({
+  Cesium3DTiles: "Cesium3DTiles",
+  PNTS: "PNTS",
+  OPC: "OPC",
+  RealityMesh3DTiles: "RealityMesh3DTiles",
+  Terrain3DTiles: "Terrain3DTiles",
+  "3MX": "3MX",
+  "3SM": "3SM",
+  CCCloudProject: "CCCloudProject",
+  CCImageCollection: "CCImageCollection",
+  CCOrientations: "CCOrientations",
+  ContextCaptureInputs: "ContextCaptureInputs",
+  ContextDetector: "ContextDetector",
+  ContextScene: "ContextScene",
+  DAE: "DAE",
+  DGN: "DGN",
+  DSM: "DSM",
+  FBX: "FBX",
+  GLB: "GLB",
+  GLTF: "GLTF",
+  KML: "KML",
+  LAS: "LAS",
+  LAZ: "LAZ",
+  LOD: "LOD",
+  LodTree: "LodTree",
+  OBJ: "OBJ",
+  OMI: "OMI",
+  OMR: "OMR",
+  Orthophoto: "Orthophoto",
+  OrthophotoDSM: "OrthophotoDSM",
+  OSGB: "OSGB",
+  OVF: "OVF",
+  OBT: "OBT",
+  PLY: "PLY",
+  PointCloud: "PointCloud",
+  S3C: "S3C",
+  ScanCollection: "ScanCollection",
+  SHP: "SHP",
+  SLPK: "SLPK",
+  SpaceEyes3D: "SpaceEyes3D",
+  STL: "STL",
+  TSM: "TSM",
+  Unstructured: "Unstructured",
+  Other: "Other",
+});
+
+ITwinPlatform.SupportedRealityDataTypes = [
```

Comment:
I shifted this supported list into the only function that uses it. `RealityDataType` cannot be made private if I want to set it as the type of the param in `createTilesetForRealityDataId` because of the way types get generated and skipped for `private`